### ISSUE TITLE
Chmod/File Upload issue in EE version 7.1.0

### ISSUE THREAD

**Issue by ymhuka** [ExpressionEngine/ExpressionEngine#2299]
Getting the following error when trying to upload a file: **_Cannot create a temp file from path: chrootmaritime-connectivity_10.jpg_**

- The name of the file is "maritime-connectivity.jpg"
- It looks like EE is trying to save different versions. I tried uploading this file 10 times when trying to troubleshoot the issue; hence the "maritime-connectivity_10.jpg"
- This issue didn't appear when updated to EE 7.0. I believe it first appeared on EE 7.1. 
- I have not modified the File Directory Structure or made any other modifications after updating to EE7.1
<img width="1115" alt="Screen Shot 2022-08-18 at 2 38 33 PM" src="https://user-images.githubusercontent.com/5405983/185538197-61dfe35f-14ac-49b3-b1d4-4f9113e37715.png">

**Comment by ymhuka** [comment]
Also, I 
<img width="404" alt="Screen Shot 2022-08-18 at 10 44 04 PM" src="https://user-images.githubusercontent.com/5405983/185538502-53e9b6b3-38b5-445b-9398-43225bc659aa.png">
noticed that EE had added "/chroot" to the "Themes Path" in "Settings."

**Comment by intoeetive** [comment]
@ymhuka  do you have a way to test using file in this PR to see if that solves your issue?
https://github.com/ExpressionEngine/ExpressionEngine/pull/2291/files

**Comment by ymhuka** [comment]
@intoeetive. Thank you for your response. We followed your directions, and I'm not sure if we did something wrong, but this is the error we got after applying the changes: "The root path is not readable." See attached. 

![image](https://user-images.githubusercontent.com/5405983/186466343-7eae9e53-3e91-49d3-9bfc-e4c0347a8927.png)


On a separate note (maybe it should be a separate bug), we ran into the following error when trying to update to 7.1.2. Seems like EE is missing "/" at the beginning of the "failed" path. Maybe it's related to the above issue, maybe not.

<img width="773" alt="Screen Shot 2022-08-24 at 11 00 43 AM" src="https://user-images.githubusercontent.com/5405983/186467020-7853768a-61fa-445f-bcfd-8db4a4150813.png">
